lib/fluent/plugin/filter_viaq_data_model.rb

```diff
@@ -381,18 +383,24 @@ def add_elasticsearch_index_name_field(tag, time, record)
           when :operations_full, :operations_prefix
             prefix = ".operations"
           when :project_full, :project_prefix
-            if (k8s = record['kubernetes']).nil?
-              log.error("record cannot use elasticsearch index name type #{ein.name_type}: record is missing kubernetes field: #{record}")
-              break
-            elsif (name = k8s['namespace_name']).nil?
-              log.error("record cannot use elasticsearch index name type #{ein.name_type}: record is missing kubernetes.namespace_name field: #{record}")
-              break
-            elsif (uuid = k8s['namespace_id']).nil?
-              log.error("record cannot use elasticsearch index name type #{ein.name_type}: record is missing kubernetes.namespace_id field: #{record}")
-              break
+            name, uuid = nil
+            unless record['kubernetes'].nil?
+              k8s = record['kubernetes']
+              name = k8s['namespace_name']
+              uuid = k8s['namespace_id']
+              if name.nil?
+                log.error("record cannot use elasticsearch index name type #{ein.name_type}: record is missing kubernetes.namespace_name field: #{record}")
```

Comment:
this will still likely be noisy until we can identify the source of the problem, if it isn't the k8s plugin